public/index.html

```diff
@@ -11,6 +11,12 @@
       content="Turnos - UNaHur"
     />
 
+    <meta property="og:title" content="Turnos UNaHur">
+    <meta property="og:image" content="%PUBLIC_URL%/apple-touch-icon.png">
```

Comment:
Vamos a probar con esta a ver si aparece. Se me hace que no andaba porque el plugin no encontraba la imagen pero que cuando lo subamos va a funcionar.

```suggestion
    <meta property="og:image" content="%PUBLIC_URL%/android-chrome-512x512.png">
```